tdrs-frontend/nginx/local/locations.conf

```diff
@@ -4,7 +4,7 @@ location = /nginx_status {
     deny all;
 }
 
-location ~ ^/(v1|admin|static/admin|static/drf-yasg|swagger|redocs) {
+location ~ ^/(v1|admin|static/admin|static/drf-yasg|static/search_indexes|swagger|redocs) {
```

Comment:
Do we need this location anymore since we don't need to use our own templates/JS? When this merges, should this be a change in my PR for the filtering?